src/emulator/storage/index.ts

```diff
@@ -67,7 +92,8 @@ export class StorageEmulator implements EmulatorInstance {
   }
 
   reset(): void {
-    this._storageLayer.reset();
```

Comment:
Not resetting the `StorageLayer`s?

Reply:
No need, StorageLayer no longer manages its own state. Instead, we pass in the bucket and file registry state in the constructor, which we reset here.